File: core/src/nl/t64/game/rpg/screens/inventory/tooltip/ShopSlotTooltipBuy.java
package nl.t64.game.rpg.screens.inventory.tooltip;

import nl.t64.game.rpg.components.party.InventoryDescription;

import java.util.List;


public class ShopSlotTooltipBuy extends ItemSlotTooltip {

    @Override
    void removeLeftUnnecessaryAttributes(List<InventoryDescription> descriptionList) {
        removeSell(descriptionList);
    }

    @Override
    void removeRightUnnecessaryAttributes(List<InventoryDescription> descriptionList) {
        removeBuy(descriptionList);
    }

}

File: core/src/nl/t64/game/rpg/screens/inventory/tooltip/ShopSlotTooltipBuy.kt
package nl.t64.game.rpg.screens.inventory.tooltip

import nl.t64.game.rpg.components.party.inventory.InventoryDescription


class ShopSlotTooltipBuy : ItemSlotTooltip() {

    override fun removeLeftUnnecessaryAttributes(descriptionList: MutableList<InventoryDescription>) {
        removeSell(descriptionList)
    }

    override fun removeRightUnnecessaryAttributes(descriptionList: MutableList<InventoryDescription>) {
        removeBuy(descriptionList)
    }

}
